llvm/test/CodeGen/NVPTX/add-rotate.ll

```diff
@@ -0,0 +1,158 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc < %s -march=nvptx64 -mcpu=sm_50 | FileCheck %s
+
+target triple = "nvptx64-nvidia-cuda"
```

Comment:
avoid using -march, it's also redundant with the triple

Reply:
Fixed